kafka/pykafka/broker.py

```diff
@@ -112,3 +114,24 @@ def request_offsets(self, partition_requests):
     def request_metadata(self, topics=[]):
         future = self.handler.request(MetadataRequest(topics=topics))
         return future.get(MetadataResponse)
+
+    ######################
+    #  Commit/Fetch API  #
+    ######################
+
+    def commit_consumer_group_offsets(self, group, preqs, retries):
+        """Commit the offsets of all messages consumed so far by this consumer
+            group with the Offset Commit/Fetch API
+
+        Based on Step 2 here https://cwiki.apache.org/confluence/display/KAFKA/Committing+and+fetching+consumer+offsets+in+Kafka
+        """
+        req = OffsetCommitRequest(group, partition_requests=preqs)
+        self.handler.request(req).get(OffsetCommitResponse)
```

Comment:
The result has nothing in it except error codes, which trigger exceptions in the protocol if present. There's no information to return.